docs/registry/configure.md

```diff
@@ -207,26 +251,30 @@ The custom model is configured as a component with the name "my-realsense".
 {{% /tab %}}
 {{% /tabs %}}
 
-### Edit the module
-
-You can also view and edit the configuration of a modular resource's underlying module from the **CONFIGURE** tab:
-
-1.  Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
-2.  Scroll to the card with the name of the module, as indicated by its page in the [registry](https://app.viam.com/registry).
+{{% /tab %}}
+{{< /tabs >}}
 
-    This is the **Registry module** card, which lists the deployed version of the module and the latest version available.
-    Here you can [configure how a module updates](#configure-version-update-management-for-a-registry-module).
+The following properties are configurable for each module:
 
-    {{<imgproc src="registry/configure/deployed-module-with-component.png" style="width: 400px" resize="500x" declaredimensions=true alt="The module card">}}
+<!--prettier-ignore-->
+| Name | Type | Required? | Description |
```

Comment:
Env still needs to also be mentioned in this table

Reply:
Yes--see slack message